pwiz_tools/Skyline/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="OneOf" version="3.0.271" targetFramework="net472" />
```

Comment:
This works on my development machine, but the build on TeamCity seems to be failing because it does not see this dll.

So far we have been adding all of the dll's that we need to:
pwiz_tools/Shared/lib

Reply:
For modern Nuget we use PackageReference, like this for Ardia in the Skyline.csproj file:
```
  <ItemGroup>
    <PackageReference Include="Microsoft.Web.WebView2">
      <Version>1.0.2420.47</Version>
    </PackageReference>
  </ItemGroup>
```

Reply:
Sounds good. Thanks for the suggestion here. I've updated OneOf to use PackageReference. It works locally. Let's see if the teamcity test can pass now

Reply:
Everything builds fine on TeamCity, but now it looks like "OneOf.dll" is not getting copied to where it needs to go.
The test "CheckForMissingResources" is failing to load "Skyline-daily.exe":
```
System.IO.FileNotFoundException: Could not load file or assembly 'OneOf, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
File name: 'OneOf, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'
```
Does anyone know how to fix something like this?